### DISCUSSION TITLE
[Config Support]: How to enable Ports?

### DISCUSSION THREAD

**Discussion by plug-it-in** [blakeblackshear/frigate#13563] [Config Support]
### Describe the problem you are having

In the help documentation under configuring go2rtc it states

Navigate to port 1984 in your browser to access go2rtc's web interface.
If using Frigate through Home Assistant, enable the web interface at port 1984.

I am using Frigate through Home Assistant. I cannot find is any instructions on how to do this. Can you please point me in the right direction to enable this port and please put this in the instructions if possible.

### Version

014.1

### Frigate config file

```yaml
NA
```

### Relevant Frigate log output

```shell
NA
```

### Relevant go2rtc log output

```shell
NA
```

### Frigate stats

```json
NA
```

### Operating system

HassOS

### Install method

HassOS Addon

### Object Detector

CPU (no coral)

**Comment by NickM-27** [comment]
Settings -> addons -> Frigate -> extra configuration -> ports -> show disabled -> enable 1984 -> save

**Comment by plug-it-in** [comment]
The only port I saw here for go2rtc was the api port I tried this. But I'm
looking for the port that means I can see the camera feed. I'm trying to
get audio working and from what I've seen the live go2rtc feed will show me
if I'm getting close.

**Reply by NickM-27** [reply]
It's all the same port, 1984

**Reply by plug-it-in** [reply]
As I said is my objective is to see the go2rtc stream either in a web page or via VLC 

I am still confused . So in this screen I hope we are talking about

![2024-09-05_14-22-05](https://github.com/user-attachments/assets/9e8a8509-7612-4442-acea-18625d6b81e8)

Where should I put 1984. Ive tried in RTSP and in go2rtc API and neither achieve what I am hoping for.

With RTSP set to 1984 and restarting I tried opening the rtsp stream in VLC using

rtsp://"useremail@gmail.com":password@homeassistant.local:1984 and this doesn't work

(useremail@gmail.com and password are my login to Home Assistant)

I note you can only specify 1984 for only one of the options in this screen which one should I use

**Reply by NickM-27** [reply]
if you want rtsp then you use port 8554 for the `RTSP Restream`

**Reply by plug-it-in** [reply]
And if I want via http ?

**Reply by NickM-27** [reply]
You use port 1984 on the go2rtc api

**Reply by plug-it-in** [reply]
So I tried 8554 using both 
rtsp://"xxxxxxxx@gmail.com":password@homeassistant.local:8554
rtsp://xxxxxx@gmail.com:password@homeassistant.local:8554

And this is the log from VLC
main: Running vlc with the default interface. Use 'cvlc' to use vlc without interface.
main: playlist is empty
live555 error: Failed to connect with rtsp://homeassistant.local:8554
main error: connection failed: Connection refused by peer
main error: connection failed: Connection refused by peer
main error: connection failed: Connection refused by peer
satip error: Failed to connect to RTSP server homeassistant.local:8554
main error: cannot resolve gmail.com%22 port 0 : No such host is known. 
access_realrtsp error: cannot connect to gmail.com%22:0
live555 error: Failed to connect with rtsp://homeassistant.local:8554
main error: connection failed: Connection refused by peer
main error: connection failed: Connection refused by peer
main error: connection failed: Connection refused by peer
main error: connection failed: Connection refused by peer
satip error: Failed to connect to RTSP server homeassistant.local:8554
main error: connection failed: Cannot assign requested address
main error: connection failed: Cannot assign requested address
access_realrtsp error: cannot connect to gmail.com:0
-- logger module stopped --

**Reply by NickM-27** [reply]
You should just use ip address not relying on mDNS
